storage/interceptors/broker_create_catalog_interceptor.go

```diff
@@ -202,6 +205,13 @@ func generateReferencePlanObject(serviceOfferingId string) *types.ServicePlan {
 	referencePlan.Ready = true
 	referencePlan.CreatedAt = time.Now()
 	referencePlan.UpdatedAt = time.Now()
+	schema, metadata, err := schemas.SchemaLoader(schemas.ReferencePlan)
```

Comment:
same as above,
can we move the entire setting of the new reference plan to the JSON file?

Reply:
so `schemas.SchemaLoader` will return a `servicePlan` object instead of `schema, metadata`